x-pack/plugins/infra/public/hooks/use_lens_attributes.ts

```diff
@@ -58,7 +56,7 @@ export const useLensAttributes = ({ dataView, ...params }: UseLensAttributesPara
   const { value, error } = useAsync(lens.stateHelperApi, [lens]);
   const { formula: formulaAPI } = value ?? {};
 
-  const attributes = useLazyRef(() => {
+  const attributes = useMemo(() => {
```

Comment:
using `useLazyRef` made the initialization of the attributes a singleton, making the attributes builder always return the same object. 

The "average" subtitle changes according to the total number of hosts, this change was never reflected due to the fact described above